sycl/source/detail/context_impl.cpp

```diff
@@ -30,7 +30,6 @@ namespace detail {
 context_impl::context_impl(const device &Device, async_handler AsyncHandler,
                            const property_list &PropList)
     : MOwnedByRuntime(true), MAsyncHandler(AsyncHandler), MDevices(1, Device),
-      MContext(nullptr),
```

Comment:
why is it removed?

Reply:
good spot. the data memeber is called `MUrContext` now but the initialization should not have been removed.

Reply:
added this back in